content/source/docs/enterprise/before-installing/reference-architecture/vmware.md

```diff
@@ -23,28 +23,25 @@ architecture.
 
 ## Infrastructure Requirements
 
+-> **Note:** This reference architecture focuses on the _Production - Mounted Disk_ operational mode.
+
 Depending on the chosen [operational
 mode](../index.html#operational-mode-decision),
 the infrastructure requirements for Terraform Enterprise range from a single virtual machine
 for demo or proof of concept installations, to multiple virtual machines
 hosting the Terraform Cloud application, PostgreSQL, and external Vault servers for
 a stateless production installation.
 
-This reference architecture focuses on the “Production - External Services”
-operational mode. If you require all of the Terraform Enterprise infrastructure to be on-prem,
-you can either deploy S3-compatible storage [such as ceph](http://ceph.com/) or select the
-“Production - Mounted Disk” option. This option will require you to specify the
-local path for the storage where PostgreSQL data will be stored and where the
-data typically written to S3 (blob) will be stored.  The assumption is this
+This reference architecture focuses on the _Production - Mounted Disk_ operational mode. This mode will require you to specify the
+local path for data storage. The assumption is this
 local path is a mounted disk from either a SAN or NAS device (or some other
 replicated storage), allowing for rapid recovery or failover. More information
 about the Mounted Disk option can be found at the end of this document.
 
 The following table provides high-level server recommendations as a guideline.
 Please note, thick provision, lazy zeroed storage is preferred. Thin
-provisioned is only recommended if you are using an external PostgreSQL database
-as well as an external vault server. Using thin provisioned disks when using
-the internal database or vault may result in serious performance issues.
+provisioned is only recommended if you are using an external PostgreSQL database and external Vault server. Using thin provisioned disks when using
```

Comment:
As Vault is an in-memory application, it shouldn't matter if the disk is thin provisioned or thick... which I should have realized when I wrote the original thing. I'd just drop the Vault bit, as we want to discourage external vault whenever possible.